CRITICAL - 2024-11-26 12:48:41 --> Trying to access array offset on value of type null
#0 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(111): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to acces...', 'C:\\xampp_8_0\\ht...', 111)
#1 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fSaveInvoice()
#2 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
ERROR - 2024-11-26 12:49:11 --> Duplicate entry 'fff' for key 'invoice_no_3'
ERROR - 2024-11-26 13:01:22 --> Duplicate entry 'aaa' for key 'invoice_no_3'
CRITICAL - 2024-11-26 13:57:37 --> Cannot use isset() on the result of an expression (you can use "null !== expression" instead)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2024-11-26 13:58:05 --> Cannot use isset() on the result of an expression (you can use "null !== expression" instead)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2024-11-26 13:58:38 --> Cannot use isset() on the result of an expression (you can use "null !== expression" instead)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2024-11-26 13:58:55 --> syntax error, unexpected variable "$set_comp_id", expecting "("
#0 C:\xampp_8_0\htdocs\invoice-generation\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp_8_0\htdocs\invoice-generation\system\Common.php(1271): CodeIgniter\View\View->render('invoice/new_inv...', Array, true)
#2 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(46): view('invoice/new_inv...', Array)
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fCreateNewInvoice()
#4 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#5 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-11-26 13:58:57 --> syntax error, unexpected variable "$set_comp_id", expecting "("
#0 C:\xampp_8_0\htdocs\invoice-generation\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp_8_0\htdocs\invoice-generation\system\Common.php(1271): CodeIgniter\View\View->render('invoice/new_inv...', Array, true)
#2 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(46): view('invoice/new_inv...', Array)
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fCreateNewInvoice()
#4 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#5 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-11-26 13:58:57 --> syntax error, unexpected variable "$set_comp_id", expecting "("
#0 C:\xampp_8_0\htdocs\invoice-generation\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp_8_0\htdocs\invoice-generation\system\Common.php(1271): CodeIgniter\View\View->render('invoice/new_inv...', Array, true)
#2 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(46): view('invoice/new_inv...', Array)
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fCreateNewInvoice()
#4 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#5 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-11-26 13:58:57 --> syntax error, unexpected variable "$set_comp_id", expecting "("
#0 C:\xampp_8_0\htdocs\invoice-generation\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp_8_0\htdocs\invoice-generation\system\Common.php(1271): CodeIgniter\View\View->render('invoice/new_inv...', Array, true)
#2 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(46): view('invoice/new_inv...', Array)
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fCreateNewInvoice()
#4 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#5 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-11-26 13:58:58 --> syntax error, unexpected variable "$set_comp_id", expecting "("
#0 C:\xampp_8_0\htdocs\invoice-generation\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp_8_0\htdocs\invoice-generation\system\Common.php(1271): CodeIgniter\View\View->render('invoice/new_inv...', Array, true)
#2 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(46): view('invoice/new_inv...', Array)
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fCreateNewInvoice()
#4 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#5 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-11-26 13:59:14 --> Unmatched ')'
#0 C:\xampp_8_0\htdocs\invoice-generation\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp_8_0\htdocs\invoice-generation\system\Common.php(1271): CodeIgniter\View\View->render('invoice/new_inv...', Array, true)
#2 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(46): view('invoice/new_inv...', Array)
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fCreateNewInvoice()
#4 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#5 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-11-26 14:00:22 --> syntax error, unexpected variable "$set_client_id", expecting "("
#0 C:\xampp_8_0\htdocs\invoice-generation\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp_8_0\htdocs\invoice-generation\system\Common.php(1271): CodeIgniter\View\View->render('invoice/new_inv...', Array, true)
#2 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(46): view('invoice/new_inv...', Array)
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fCreateNewInvoice()
#4 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#5 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-11-26 14:06:35 --> Array callback must have exactly two elements
#0 C:\xampp_8_0\htdocs\invoice-generation\system\View\View.php(230): include()
#1 C:\xampp_8_0\htdocs\invoice-generation\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\xampp_8_0\htdocs\invoice-generation\system\Common.php(1271): CodeIgniter\View\View->render('invoice/new_inv...', Array, true)
#3 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(46): view('invoice/new_inv...', Array)
#4 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fCreateNewInvoice()
#5 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#6 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2024-11-26 14:38:51 --> Column 'invoice_receiver_id' cannot be null
ERROR - 2024-11-26 14:41:01 --> Duplicate entry '123' for key 'invoice_no_3'
ERROR - 2024-11-26 14:42:43 --> Column 'receiver_comp_name' cannot be null
ERROR - 2024-11-26 14:43:06 --> Column 'receiver_comp_name' cannot be null
ERROR - 2024-11-26 14:45:00 --> Column 'receiver_comp_name' cannot be null
ERROR - 2024-11-26 14:45:20 --> Column 'receiver_comp_name' cannot be null
CRITICAL - 2024-11-26 15:11:57 --> Cannot use [] for reading
#0 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fSaveInvoice()
#1 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#2 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2024-11-26 15:15:21 --> Undefined array key 1
#0 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(147): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp_8_0\\ht...', 147)
#1 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fSaveInvoice()
#2 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
ERROR - 2024-11-26 15:51:12 --> Unknown column 'is_active' in 'where clause'
CRITICAL - 2024-11-26 15:51:12 --> Call to a member function getResultArray() on bool
#0 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(135): App\Models\Master_model->getRecords('inv_invoice_det...', Array, 'id')
#1 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fSaveInvoice()
#2 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
ERROR - 2024-11-26 15:52:29 --> Duplicate entry '45' for key 'PRIMARY'
ERROR - 2024-11-26 15:52:29 --> Duplicate entry '46' for key 'PRIMARY'
ERROR - 2024-11-26 15:53:35 --> Duplicate entry '45' for key 'PRIMARY'
ERROR - 2024-11-26 15:53:35 --> Duplicate entry '46' for key 'PRIMARY'
ERROR - 2024-11-26 15:53:43 --> Duplicate entry '45' for key 'PRIMARY'
ERROR - 2024-11-26 15:53:43 --> Duplicate entry '46' for key 'PRIMARY'
CRITICAL - 2024-11-26 16:07:28 --> Undefined variable $ids
#0 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(154): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp_8_0\\ht...', 154)
#1 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fSaveInvoice()
#2 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2024-11-26 16:08:45 --> Undefined array key 2
#0 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(143): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp_8_0\\ht...', 143)
#1 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fSaveInvoice()
#2 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2024-11-26 16:54:46 --> Undefined variable $aInvoiceData
#0 C:\xampp_8_0\htdocs\invoice-generation\app\Views\dashboard\index.php(89): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp_8_0\\ht...', 89)
#1 C:\xampp_8_0\htdocs\invoice-generation\system\View\View.php(230): include('C:\\xampp_8_0\\ht...')
#2 C:\xampp_8_0\htdocs\invoice-generation\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp_8_0\htdocs\invoice-generation\system\Common.php(1271): CodeIgniter\View\View->render('dashboard/index', Array, true)
#4 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Dashboard.php(20): view('dashboard/index', Array)
#5 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Dashboard->fBoardDashboard()
#6 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#7 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2024-11-26 16:55:31 --> Unknown column 'deleted' in 'where clause'
CRITICAL - 2024-11-26 16:55:31 --> Call to a member function getResultArray() on bool
#0 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Dashboard.php(20): App\Models\Master_model->getRecords('inv_clients', Array)
#1 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Dashboard->fBoardDashboard()
#2 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
ERROR - 2024-11-26 17:07:22 --> Unknown column 'is_active' in 'where clause'
CRITICAL - 2024-11-26 17:07:22 --> Call to a member function getResultArray() on bool
#0 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(54): App\Models\Master_model->getRecords('inv_invoice_mas...', Array, 'invoice_no')
#1 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->FupdateInvoice('6')
#2 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2024-11-26 17:12:57 --> syntax error, unexpected token "[", expecting identifier or variable or "{" or "$"
#0 C:\xampp_8_0\htdocs\invoice-generation\system\Autoloader\Autoloader.php(273): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp_8_0\\ht...')
#1 C:\xampp_8_0\htdocs\invoice-generation\system\Autoloader\Autoloader.php(247): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(885): class_exists('\\App\\Controller...', true)
#4 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(421): CodeIgniter\CodeIgniter->startController()
#5 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2024-11-26 17:17:56 --> Column 'receiver_comp_name' cannot be null
CRITICAL - 2024-11-26 17:54:03 --> Undefined array key "GetInvoiceData"
#0 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(266): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp_8_0\\ht...', 266)
#1 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fShareInvoice()
#2 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2024-11-26 17:56:23 --> Undefined array key 0
#0 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(266): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp_8_0\\ht...', 266)
#1 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fShareInvoice()
#2 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2024-11-26 19:38:49 --> syntax error, unexpected token "return"
#0 C:\xampp_8_0\htdocs\invoice-generation\system\Autoloader\Autoloader.php(273): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp_8_0\\ht...')
#1 C:\xampp_8_0\htdocs\invoice-generation\system\Autoloader\Autoloader.php(247): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(885): class_exists('\\App\\Controller...', true)
#4 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(421): CodeIgniter\CodeIgniter->startController()
#5 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-11-26 20:06:07 --> Class "Mpdf\Mpdf" not found
#0 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fShareInvoice()
#1 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#2 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2024-11-26 20:14:07 --> Invalid file: html_to_pdf.php
#0 C:\xampp_8_0\htdocs\invoice-generation\system\View\View.php(213): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('html_to_pdf.php')
#1 C:\xampp_8_0\htdocs\invoice-generation\system\Common.php(1271): CodeIgniter\View\View->render('html_to_pdf', Array, true)
#2 C:\xampp_8_0\htdocs\invoice-generation\app\Controllers\Invoices.php(279): view('html_to_pdf', Array)
#3 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fShareInvoice()
#4 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#5 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-11-26 20:14:27 --> Call to undefined method PhpOffice\PhpSpreadsheet\Writer\Pdf\Mpdf::WriteHTML()
#0 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(928): App\Controllers\Invoices->fShareInvoice()
#1 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Invoices))
#2 C:\xampp_8_0\htdocs\invoice-generation\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp_8_0\htdocs\invoice-generation\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
